app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -264,6 +264,12 @@ def raw_connect_v4(options = {})
       # Get the timeout from the configuration:
       timeout, = ems_timeouts(:ems_redhat, options[:service])
 
+      # The constructor of the SDK expects a list of certificates, but that list can't be empty, or contain only 'nil'
+      # values, so we need to check the value passed and make a list only if it won't be empty. If it will be empty then
+      # we should just pass 'nil'.
+      ca_certs = options[:ca_certs]
+      ca_certs = [ca_certs] if ca_certs
```

Comment:
so `options[:ca_certs]` is not an array already? Just read the code and it seems to be a concatenated String of certs... well :)

```ruby
:ca_certs => options[:ca_certs].present? ? Array(options[:ca_certs] : nil
```

because
```ruby
[8] pry(main)> Array(nil)
=> []
[9] pry(main)> Array([])
=> []
[10] pry(main)> [].present?
=> false
[11] pry(main)> ''.present?
=> false
[12] pry(main)> nil.present?
=> false
```